src/zenml/zen_stores/schemas/code_repository_schemas.py

```diff
@@ -109,13 +109,13 @@ def to_model(self, hydrate: bool = False) -> "CodeRepositoryResponse":
             user=self.user.to_model() if self.user else None,
             source=json.loads(self.source),
             logo_url=self.logo_url,
+            created=self.created,
```

Comment:
Why was this done?

Reply:
You caught me. This was a minor, unrelated bug that I fixed here for @Cahllagerfeld . Too small to open a separate PR for it :)

Reply:
Haha, coolcool!